hack/update-vendor.sh

```diff
@@ -33,7 +33,7 @@ export LANG=C
 export LC_ALL=C
 # Detect problematic GOPROXY settings that prevent lookup of dependencies
 if [[ "${GOPROXY:-}" == "off" ]]; then
-  kube::log::error "Cannot run hack/update-vendor.sh with \$GOPROXY=off"
+  kube::log::error "Cannot run hack/update-vendor.sh with \$GOPROXY=off" >&22
```

Comment:
does calling `kube::log::error` even make sense with this? (it redirects to >2 internally)

Reply:
this happens before we've defined 22, right? also, kube::log::error redirects to >&2 internally... should this line stay as it was?

Reply:
good point - This should be 2>&1

Reply:
if it's before all our exec and `set -x` stuff below, can't it just remain `kube::log::error`?